src/rpc/server.cpp

```diff
@@ -31,12 +31,12 @@ static std::string rpcWarmupStatus GUARDED_BY(cs_rpcWarmup) = "RPC server starte
 static RPCTimerInterface* timerInterface = nullptr;
 /* Map of name to timer. */
 static std::map<std::string, std::unique_ptr<RPCTimerBase> > deadlineTimers;
+static std::shared_ptr<int> g_active_commands_counter = std::make_shared<int>(0);
```

Comment:
This is temporary, just for concept ACK.

Reply:
Yeah. I suggest using a manual increment rather then the `use_count()`